src/Sulu/Bundle/ContentBundle/Controller/NodeController.php

```diff
@@ -776,6 +776,17 @@ public function postTriggerAction($uuid, Request $request)
                     $this->getDocumentManager()->removeDraft($data, $language);
                     $this->getDocumentManager()->flush();
                     break;
+                case 'restore':
```

Comment:
Should we somehow disable this action if versioning is disabled? I'd say we should, the question is how. Put all of the code in this `case` in a `if`, and let the code fall through, or throw the exception in this branch? As `Unrecognized action` or something like `versioning is disabled`?